### PR TITLE
Fix: Fixed an issue where slightly dragging a tab would create a new window

### LINKED ISSUE #11081: Bug: Tab becomes its own window if clicked on a bit too long
### Description

sometimes if i click on a tab and dont release it fast enough it will move the tab into a new window, according to Josh-65 in the discord this could be cause the drag sensitivity is too high and only encountered it with Files.

### Steps To Reproduce

open several tabs.
Click on the tabs with greater and greater delay between press down and release, eventually one should pop out as its own window.

### Steps to fix

either implement an option to disable pop out when dragged or figure out why this is happening in the first place.

### Files Version

Version: 2.4.21.0 OS Version: 10.0.19045.2486

### Windows Version

Win 10 22H2 build 19045.2486

### Relevant Assets/Logs

[debug.log](https://github.com/files-community/Files/files/10504410/debug.log)

### PR DESCRIPTION
**Pull request by yaira2**
**Resolved / Related Issues**
- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Closes #11081...

**Validation**
How did you test these changes?
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [x] Are there any other steps that were used to validate these changes?
   1. Create multiple tabs
   2. Drag a tab a few pixels and confirm it doesn't create a new window (you can also test with the stable version to see the previous behavior)
   3. Drag a tab a little further and confirm that it creates a new window

**Screenshots (optional)**
Add screenshots here.

### CODE REVIEW

**Comment by yaira2** [comment]
@Josh65-2201 can you confirm if this is working as expected

File: src/Files.App/UserControls/TabBar/TabBar.xaml.cs [hunk 1]
```diff
@@ -211,14 +221,20 @@ private void TabView_TabDragCompleted(TabView sender, TabViewTabDragCompletedEve
...
-			if (sender.TabItems.Count == 1)
-			{
+			InteropHelpers.GetCursorPos(out var droppedPoint);
+			var droppedTime = DateTimeOffset.UtcNow;
+			var dragTime = droppedTime - dragStartTime;
+			var dragDistance = Math.Sqrt(Math.Pow((dragStartPoint.X - droppedPoint.X), 2) + Math.Pow((dragStartPoint.Y - droppedPoint.Y), 2));
+
+			if (sender.TabItems.Count == 1 ||
+				dragTime.TotalSeconds < 1 &&
+				dragDistance < 100)
```
Comment (d2dyno1) [inline src/Files.App/UserControls/TabBar/TabBar.xaml.cs:231]: The second condition (after `||`) should be wrapped with a parenthesis or replace `&&` with `||`

File: src/Files.App/UserControls/TabBar/TabBar.xaml.cs [hunk 2]
```diff
@@ -136,6 +142,10 @@ private void TabView_TabDragStarting(TabView sender, TabViewTabDragStartingEvent
 			var tabViewItemArgs = (args.Item as TabBarItem).NavigationParameter;
 			args.Data.Properties.Add(TabPathIdentifier, tabViewItemArgs.Serialize());
 			args.Data.RequestedOperation = DataPackageOperation.Move;
+
+			InteropHelpers.GetCursorPos(out var point);
+			dragStartPoint = point;
```
Comment (d2dyno1) [inline src/Files.App/UserControls/TabBar/TabBar.xaml.cs:147]:
```suggestion
			InteropHelpers.GetCursorPos(out dragStartPoint);
```

**Comment by Josh65-2201** [comment]
Yes it fixed it for me.

File: src/Files.App/UserControls/TabBar/TabBar.xaml.cs (hunk 1 above)
Comment (yaira2) [reply src/Files.App/UserControls/TabBar/TabBar.xaml.cs]: I pushed a commit with this change

**Pull request merged**